praw/config.py

```diff
@@ -100,9 +100,9 @@ def _initialize_attributes(self):
                       ['comment', 'message', 'redditor', 'submission',
                        'subreddit']}
 
-        for attribute in ('client_id', 'client_secret', 'http_proxy',
-                          'https_proxy', 'redirect_uri', 'refresh_token',
-                          'password', 'user_agent', 'username'):
+        for attribute in ('client_id', 'client_secret', 'redirect_uri',
```

Comment:
Great catch! I wasn't sure what you had meant about reading in these values. Thanks for the PR.